pkg/antctl/antctl.go

```diff
@@ -161,6 +161,19 @@ var CommandList = &commandList{
 			commandGroup:        flat,
 			transformedResponse: reflect.TypeOf(agentinfo.AntreaAgentInfoResponse{}),
 		},
+		{
+			use:   "health-check",
+			short: "Return the health status of Antrea agent",
+			long:  "Implement all health checkers installed and return the health status",
+			agentEndpoint: &endpoint{
+				nonResourceEndpoint: &nonResourceEndpoint{
+					path:     "/healthcheck",
+					isSingle: true,
+				},
+			},
+			commandGroup:        flat,
+			transformedResponse: reflect.TypeOf(healthcheck.AntreaAgentHealthCheckResponse{}),
```

Comment:
I know `/healthz` just returns plain instead of json, can the CLI framework adapt it? or it has to be a json?

Reply:
This needs to be checked.